frontend/package.json

```
@@ -42,6 +42,7 @@
     "dotenv": "16.4.5",
     "eslint": "9.10.0",
     "eslint-plugin-cypress": "3.5.0",
+    "flag-icons": "^7.2.3",
```

```suggestion
    "flag-icons": "7.2.3",
```

https://docs.rotki.com/contribution-guides/vue-typescript.html#versions